helm/chart/router/values.yaml

```diff
@@ -50,6 +45,16 @@ extraEnvVarsSecret: ''
 extraVolumes: []
 extraVolumeMounts: []
 
+## @param extraConfigmaps Array to mount extra ConfigMaps to the router
```

Comment:
This is a good idea but with extra volumes & volume mounts you can just do something like [this](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#populate-a-volume-with-data-stored-in-a-configmap)

Reply:
Hmm. I'm trying to decide if there's any point adding this `extraConfigMaps` or if we can do it all via `Volumes/VolumeMounts`. My motivation is to make it clear and simple to add `rhai` scripts to your pod. If the `extraConfigMaps` delivers nothing new, I'd be happy to ditch that part of the PR and just use `Volumes` as you linked to.